src/mork/edx/mixins.py

```diff
@@ -1,20 +1,38 @@
 """Module for defining sessions classes."""
 
-from datetime import datetime, timedelta
+from datetime import datetime
 from typing import Optional
 
-from sqlalchemy import select
+from sqlalchemy import distinct, select
 from sqlalchemy.orm import Session, load_only
+from sqlalchemy.sql.functions import count
 
 
 class UserMixin:
     """Mixin that adds convenience methods for the User operations."""
 
+    @classmethod
+    def get_inactives_count(
```

Comment:
I do not find this name easy to read at first sight. What about: 
```suggestion
    def get_inactive_users_count(
```
OR
```suggestion
    def count_inactives(
```

Reply:
You're right! Switched to `get_inactive_users` and `get_inactive_users_count`